projects/coordinator/pom.xml

```diff
@@ -141,5 +141,15 @@
       <artifactId>slf4j-jdk14</artifactId>
       <scope>runtime</scope>
     </dependency>
+      <dependency>
```

Comment:
looks like the indent is messed up here (did you use tabs/spaces by any chance)

please also copy the comment from other pom.xml that explains that below this line is test dependencies.